### PR TITLE
add Faraday RBS

### PR DESCRIPTION
**Pull request by tatematsu-k**
Add RBS file of [Faraday](https://github.com/lostisland/faraday).

I don't know much about Faraday, so added RBS of frequently used methods.
Please check it.

### CODE REVIEW

File: gems/faraday/2.5/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# manifest.yaml describes dependencies which do not appear in the gemspec.
+# If this gem includes such dependencies, comment-out the following lines and
+# declare the dependencies.
+# If all dependencies appear in the gemspec, you can remove this file.
```
Comment (pocke) [inline gems/faraday/2.5/manifest.yaml:4]: Could you remove this file?

**Review by pocke — COMMENTED** [review]
Just confirmation, did you mean "RBS" but not ERB?

File: gems/faraday/2.5/manifest.yaml (hunk 1 above)
Comment (tatematsu-k) [reply gems/faraday/2.5/manifest.yaml]: sure. remove on this commit.
https://github.com/ruby/gem_rbs_collection/pull/198/commits/b6f2e9a4da70affb82499645ac8ad35781052547

**Comment by tatematsu-k** [comment]
> Just confirmation, did you mean "RBS" but not ERB?

It's typo. RBS is correct!
fix PR title and description 🙏

**Comment by tatematsu-k** [comment]
Merged main for resolving conflict.
https://github.com/ruby/gem_rbs_collection/pull/198/commits/cc36e2835928bd686261ff4aafeb69ec643cd6f2

**Review by pocke — APPROVED** [review]
Thanks!

**Pull request merged**